include/itkMaxPhaseCorrelationOptimizer.hxx

```diff
@@ -151,7 +151,7 @@ MaxPhaseCorrelationOptimizer< TRegistrationMethod >
 
   // supress trivial zero solution
   const typename ImageType::IndexType oIndex = lpr.GetIndex();
-  const PixelType                     zeroDeemphasis1 = std::max< PixelType >( 1.0, m_ZeroSuppression / 2.0 );
+  const PixelType zeroDeemphasis1 = std::max< PixelType >( 1.0, m_ZeroSuppression / 2.0 );
   for ( i = 0; i < maxs.size(); i++ )
```

Comment:
i should have local scope to the for loop.  Not defined on line 116.  The variable on line 116 should be different than the variable at line 155.

Reply:
OK I will split this into a style commit and an ENH